source/containers.tex

```diff
@@ -1710,21 +1710,24 @@
 \tcode{c} denotes a possibly \keyword{const} value of type \tcode{X::key_compare};
 \item
 \tcode{kl} is a value such that \tcode{a} is partitioned\iref{alg.sorting}
-with respect to \tcode{c(r, kl)}, with \tcode{r} the key value of \tcode{e}
```

Comment:
I appreciate the heavy maths jargon here, and even though it resonates on a personal level, I think it's maybe a bit unnecessary in a general text. What do you think of changing this to the more normal "where `r` is the key value of `e` and `e` is in [or "an element of"] `a`"?

Reply:
Well, the existing text already has the phrase; we're just replacing "r" with "x" and making sure the phrase is duplicated everywhere "x" is used.

The later bullets already contain a "with" phrase, so continuing with "where" is a bit inconsistent.

Reply:
(If we want to improve clarity here, adding "all" would probably also be helpful.)

Reply:
It's the "consistency with a bad idea" thing... OK, let's make the change as is, and maybe we can rephrase everything afterwards.

Reply:
(Note to selves: revisit this.)